src/main/java/core/basesyntax/service/operationhandler/OperationHandler.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.service.operationhandler;
+
+public interface OperationHandler {

```

Comment:
move all handlers to package strategy
create "impl" package in the strategy and move the entire implementation